src/routes/(app)/settings/+page.server.ts

```diff
@@ -44,46 +38,56 @@ export const actions: Actions = {
     const { user, prisma } = locals;
     const form = await request.formData();
     if (!user) return fail(401, { form });
```

Comment:
this should probably use zod and superforms to handle type validation, like we do in other form inputs